sdk/runtime/runner.go

```diff
@@ -45,7 +45,14 @@ func Invoke(tc func(*RunEnv) error) {
 	ctx, cancel := context.WithCancel(context.Background())
 
 	setupHTTPListener(runenv)
-	metricsDoneCh := setupMetrics(ctx, runenv)
+	pushGatewayEnabled := false
```

Comment:
Raising this to attention of the team, in case anyone is still using it - @coryschwartz @raulk

Reply:
yep, this means the gateway is currently disabled, so we should either remove this or make it configurable.